operations/countries.md

```diff
@@ -10,84 +11,96 @@ Returns all countries supported by the API.
-    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
-    "Client": "Sample Client 1.0.0"
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0"
 }
 ```
 
 | Property | Type | Contract | Description |
-| :--- | :--- | :--- | :--- |
-| `ClientToken` | string | required | Token identifying the client application. |
-| `AccessToken` | string | required | Access token of the client application. |
+| :-- | :-- | :-- | :-- |
 | `Client` | string | required | Name and version of the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `ClientToken` | string | required | Token identifying the client application. |
 
 ### Response
 
 ```javascript
 {
-    "Countries": [
-        {
-            "Code": "GB",
-            "EnglishName": "United Kingdom of Great Britain and Northern Ireland"
-        },
-        {
-            "Code": "US",
-            "EnglishName": "United States of America"
-        }
-    ],
-    "CountrySubdivisions": [
-        {
-            "Code": "AU-NSW",
-            "CountryCode": "AU",
-            "EnglishName": "New South Wales"
-        },
-        {
-            "Code": "AU-QLD",
-            "CountryCode": "AU",
-            "EnglishName": "Queensland"
-        }
-    ],
-    "CountryAlliances": [
-        {
-            "Code": "SCHENGEN",
-            "EnglishName": "Schengen Area",
-            "CountryCodes":[
-                "AT",
-                "BE",
-                "CZ",
-                ...
-            ]
-        }
-    ]
+  "Countries": [
+    {
+      "Code": "GB",
+      "EnglishName": "United Kingdom of Great Britain and Northern Ireland"
+    },
+    {
+      "Code": "US",
+      "EnglishName": "United States of America"
+    }
+  ],
+  "CountrySubdivisions": [
+    {
+      "Code": "AU-NSW",
+      "CountryCode": "AU",
+      "EnglishName": "New South Wales"
+    },
+    {
+      "Code": "AU-QLD",
+      "CountryCode": "AU",
+      "EnglishName": "Queensland"
+    }
+  ],
+  "CountryAlliances": [
+    {
+      "Code": "SCHENGEN",
+      "EnglishName": "Schengen Area",
+      "CountryCodes": [
+        "AT",
+        "BE",
+        "CZ",
+        "..."
+      ]
+    }
+  ]
 }
 ```
 
 | Property | Type | Contract | Description |
-| :--- | :--- | :--- | :--- |
+| :-- | :-- | :-- | :-- |
 | `Countries` | array of [Country](#country) | required | The supported countries. |
 | `CountrySubdivisions` | array of [Country subdivision](#country-subdivision) | required | The supported country subdivisions. |
-| `CountryAlliances` | array of [Country alliance](#country-alliance) | required | The supported country alliances. |
+| `CountryRules` | array of [Country rules](#country-rules) | required |  |
```

Comment:
I see `CountryAlliances` is being replaced by `CountryGroups`, is that right? I presume the documentation was incorrect before, otherwise it looks like a deprecation.

Reply:
The documentation was incorrect in this case. Since examples are injected through separate repository, I would like to merge this change with outdated example and fix it later.